### PR TITLE
ref(py3): Use byte type for binary secret key in tests

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: tests/sentry/models/test_organizationmember.py [hunk 1]
```diff
@@ -22,8 +22,8 @@ def test_legacy_token_generation_unicode_key(self):
         member = OrganizationMember(id=1, organization_id=1, email="foo@example.com")
         with self.settings(
             SECRET_KEY=(
-                "\xfc]C\x8a\xd2\x93\x04\x00\x81\xeak\x94\x02H"
-                "\x1d\xcc&P'q\x12\xa2\xc0\xf2v\x7f\xbb*lX"
+                b"\xfc]C\x8a\xd2\x93\x04\x00\x81\xeak\x94\x02H"
+                b"\x1d\xcc&P'q\x12\xa2\xc0\xf2v\x7f\xbb*lX"
             )
```
Comment (wedamija) [inline tests/sentry/models/test_organizationmember.py:27]: Is this value actually going to be bytes? We get it from `bootstrap_options`, which I think gets it from the yml config file.

Comment (EvanPurkhiser) [reply tests/sentry/models/test_organizationmember.py:27]: https://github.com/getsentry/sentry/blob/90e55f88fffed41079d3f2268cb77e8d02f1c850/src/sentry/models/organizationmember.py#L198

We force it to bytes here

Comment (EvanPurkhiser) [reply tests/sentry/models/test_organizationmember.py:27]: I could decode this back to a string here if we wanted

Comment (wedamija) [reply tests/sentry/models/test_organizationmember.py:27]: Hmm, I guess it's fine since it's only used in that one place.

**Review by wedamija — APPROVED** [review]

**Pull request merged**